reference/docs-conceptual/learn/remoting/SSH-Remoting-in-PowerShell-Core.md

```diff
@@ -105,7 +105,7 @@ an SSH subsystem to host a PowerShell process on the remote computer. And, you m
 
    Optionally, enable key authentication:
 
-   ```
+   ```Output
```

Comment:
This is not output. This is another setting within the configuration file.

Reply:
Fixed